webapp/src/ts/components/filters/date-filter/date-filter.component.ts

```diff
@@ -33,7 +33,6 @@ export class DateFilterComponent implements OnInit, OnDestroy, AfterViewInit {
   @Input() disabled;
   @Input() isStartDate;
   @Input() fieldId;
-  // eslint-disable-next-line @angular-eslint/no-output-native
```

Comment:
Not 100% sure what was going on here, but it was apparently related to some changes that had leaked into the webapp `package-lock.json` file.  Reverting your changes to that fail seems to have resolved the linting problem here.